business-registry-dashboard/app/stores/affiliations.ts

```diff
@@ -302,11 +349,85 @@ export const useAffiliationsStore = defineStore('brd-affiliations-store', () =>
       await createAffiliation({ businessIdentifier })
       toast.add({ title: `${businessIdentifier} successfully added to your list` }) // add success toast
       await loadAffiliations() // reload affiliated entities
-    } catch (err) {
-      const e = err as FetchError
-      console.error('Error adding business: ', e.data)
+    } catch (error) {
+      logFetchError(error, 'Error adding business')
+      const e = error as FetchError
+      const msg = e.data?.message ?? ''
+      toast.add({ title: t('toast.unableToAddBusiness'), description: msg })
+    }
+  }
+
+  /* Internal function for sorting affiliations / entities by invites. */
+  function sortEntitiesByInvites (affiliatedEntities: Business[]): Business[] {
+    // bubble the ones with the invitations to the top
+    affiliatedEntities?.sort((a, b) => {
+      if (a.affiliationInvites && !b.affiliationInvites) {
+        return -1
+      }
+      if (!a.affiliationInvites && b.affiliationInvites) {
+        return 1
+      }
+      return 0
+    })
+    return affiliatedEntities
+  }
+
+  // need to get business identifier in the args
+  async function resendAffiliationInvitation (event: Business) {
+    const invite = event.affiliationInvites?.[0]
+    // let invitationId = ''
+    const invitationId = ''
+
+    // if (this.base64Token && this.base64OrgName) { // TODO: implement base 64 token from magic link
+    //   const base64TokenObject = this.base64Token.split('.')[0]
+    //   const decodedToken = Base64.decode(base64TokenObject)
+    //   const token = JSON.parse(decodedToken)
+    //   invitationId = token.id
+    // }
+
+    try {
+      const affiliationInvitationId = invitationId || invite?.id || ''
+      await $authApi(`/affiliationInvitations/${affiliationInvitationId}`, {
+        method: 'PATCH',
+        body: {} // empty body required
+      })
+
+      if (invite?.recipientEmail === undefined) { // show toast if no email in the event object
+        toast.add({ title: t('form.manageBusiness.toast.emailSent') })
+      } else { // else open confirmation/instructions modal
+        brdModal.openAuthEmailSent(invite.recipientEmail)
+      }
+    } catch (error) {
+      logFetchError(error, 'Error resending affiliation invitation')
+      const e = error as FetchError
       const msg = e.data?.message ?? ''
-      toast.add({ title: 'Unable to add business', description: msg })
+      toast.add({ title: t('toast.errorResendingAffInvite'), description: msg })
+    }
+  }
+
+  async function deletePendingInvitations (businessIdentifier: string) {
```

Comment:
delete all pending invitations for a given identifier/accountid, required when making a request for a business that already has a pending invite